airbyte-integrations/connectors/source-file/integration_tests/sample_files/formats/pickle/configured_catalog_pickle.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
if we don't support parquet/orc/pickle can we remove them and any catalogs/sample files that are not supported?

Reply:
@sherifnada we support parquet and pickle

Reply:
Pickle works if it contains DataFrame. Because read_pickle() method returns same type as object stored in file. And discover and read methods of client expect for DataFrame.